docs/relnotes.md

```diff
@@ -7,18 +7,62 @@ hide:
 
 This is the list of changes to Taipy releases as they were published.
 
-## Community edition: 1.2 (In progress)
+## Community edition: 2.0 (In progress)
```

Comment:
Why the (in progress) segment?

Reply:
Cause it s not delivered yet. But I can remove it.